schema/iea43_wra_data_model.schema.json

```diff
@@ -992,6 +992,28 @@
                           "false (for no flow corrections applied)"
                         ]
                       },
+                      "logger_stated_device_datum_plane_height": {
```

Comment:
Forgot the trailing `_m`.

This is what is in the Vertical Profiler Properties

```JSON
"device_datum_plane_height_m": {
                  "type": [
                    "number",
                    "null"
                  ],
"title": "Device Datum Plane Height [m]",
"description": "The datum plane height of the remote sensing device where the datum feature is defined here: http://data.windenergy.dtu.dk/ontologies/view/ontolidar/en/page/DatumFeature . For lidars the datum feature is usually the window and for sodars it is usually the base of the device. These datum plane heights are also usually referred to as above ground level however it may be above sea level or above a platform level.",
```